documentation/book/proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0.adoc

```diff
@@ -0,0 +1,61 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0-{context}']
+= Upgrading the Cluster Operator from 0.10.0 to 0.11.0
+
+This procedure will describe how to upgrade a Cluster Operator deployment from version 0.10.0 to version 0.11.0.
+The availability of Kafka clusters managed by the Cluster Operator is not affected by the upgrade operation.
+
+.Prerequisites 
+
+* An existing version 0.10.0 Cluster Operator deployment to be upgraded.
+
+.Procedure
+
+. Backup the existing Cluster Operator resources.
++
+ifdef::Kubernetes[]
+On {KubernetesName} use `kubectl get`:
++
+----
+kubectl get all -l app=strimzi > strimzi-backup.yaml
+----
++
+endif::Kubernetes[]
+On {OpenShiftName} use `oc get`:
++
+----
+oc get -l all app=strimzi > strimzi-backup.yaml
+----
+
+. Update the Cluster Operator. 
+You will need to modify the installation files according to the namespace the Cluster Operator is running in.
```

Comment:
Please specify which installation files need to be modified.

Reply:
@laidan6000 I think that this is the same statement as in the existing chapter for 0.9.0->0.10.0 upgrade or what we have in the installation chapters. There is a command below it which takes care of this. So I'm not sure we need to list them.